_security/configuration/yaml.md

```diff
@@ -284,6 +281,8 @@ _meta:
 
 ## roles_mapping.yml
 
+Typically found at `<OPENSEARCH_HOME>/config/opensearch-security/roles_mapping.yml`.
```

Comment:
This file contains any role mappings required for your security configuration. You can find the `role_mapping.yml` file in `<OPENSEARCH_HOME>/config/opensearch-security/roles_mapping.yml`.